trillian/ctfe/configpb/config.proto

```diff
@@ -76,6 +76,9 @@ message LogConfig {
   // checked against the current time during the validation of submissions.
   // This will cause expired certificates to be rejected.
```

Comment:
(pre-existing) These conditions cover not-yet-valid certs as well as expired certs -- dunno if it's worth updating the comment wording to cover both cases?

Reply:
Done.